.travis.yml

```diff
@@ -86,6 +86,10 @@ matrix:
       arch: amd64
       env: TR_ARCH=armv7-cross
       dist: bionic
+    - os: linux
+      arch: mips64
```

Comment:
This does not work. Travis does not provide `mips64`. Travis should just fail but this test is running on `x86_64` if you look at the log files. You need to look at how we used to run docker with qemu emulation for aarch64, ppc64le and s390x before Travis had native support for those architectures.

Reply:
Or you could try to install a cross compiler into the Travis environment and provide cross-compilation test.

Reply:
I am sorry. I am not understand,can you tall me how to try to install a cross compiler into the Travis environment? or I should add a file like Dockfile.mips.hdr ?

Reply:
Please have a look how it is done for arm here https://github.com/checkpoint-restore/criu/blob/criu-dev/scripts/build/Dockerfile.armv7-cross . This uses a cross compiler.

The other way is to use docker and qemu to emulate another architecture:
https://github.com/checkpoint-restore/criu/blob/v3.13/scripts/build/Dockerfile.aarch64.hdr
https://github.com/checkpoint-restore/criu/blob/v3.13/scripts/build/binfmt_misc

The whole binfmt, qemu, docker combination has been removed from the criu-dev branch as all other CRIU architectures (arm, aarch64, s390x, ppc64le, x86_64) are natively supported by Travis. You have to decide what is better for mips: cross-compiler or the qemu docker combination.

Reply:
I have add scripts/build/Dockerfile.mips64el.hdr. but I don't know how to make it work